google/earthenterprise
# #1640: Bundle and Upgrade Swig

Bundles a newer version of swig to complete the C++11 migration. 

To Test:
1. Build portable.
2. Verify that it builds and works.
3. Build OpenGEE.
4. Verify that the build completes and fusion and server are both working.
